src/data/openthaigpt_pretraining_data/web_crawls_mfa/crawl_top_news.py

```diff
@@ -0,0 +1,130 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import time
+
+ROOT = 'https://www.mfa.go.th'
+TOP_STORIES_URL = 'https://www.mfa.go.th/th/page/%E0%B8%82%E0%B9%88%E0%B8%B2%E0%B8%A7%E0%B9%80%E0%B8%94%E0%B9%88%E0%B8%99?menu=5d5bd3d815e39c306002aac4'
+UNWANTED_CLASSES = [('div','d-inline-block'),('div','pt-3 col')]
+DIV_TAG = 'div'
+P_TAG = 'p'
+A_TAG = 'a'
+DATE_CLASS = 'date'
+INFO_CLASS = 'p-3 col-md-4'
+DETAIL_CLASS = 'ContentDetailstyled__ContentDescription-sc-150bmwg-4 jWrYsI mb-3'
+
+def process_title_date(response):
```

Comment:
Input should be text_response because of this function you use only `response.content`.